rules/fastqc/Snakefile

```diff
@@ -0,0 +1,44 @@
+     
+rule fastqc: 
+    input:
+        forward="{prefix}%s%s/{sample}_R1.fastq.gz" % (config['dirs']['intermediate'], config['stepnames']['rejoin_samples']),
+        reverse="{prefix}%s%s/{sample}_R2.fastq.gz" % (config['dirs']['intermediate'], config['stepnames']['rejoin_samples']),
+    output: 
+        determinedforward="{prefix}%s%s/determined/fastqc/{sample, .*?}_R1.fastqc.html" % (config['dirs']['intermediate'], config['stepnames']['Fastqc']),
+        determinedreverse="{prefix}%s%s/determined/fastqc/{sample, .*?}_R2.fastqc.html" % (config['dirs']['intermediate'], config['stepnames']['Fastqc']),
+    log:
+        "{prefix}%s%s/{sample}.log" % (config['dirs']['logs'], config['stepnames']['Fastqc'])
+    conda: 
+        "envs/qc.yml"
+    threads: 
+        "4"
+    params:
+        undeterminedforward="{prefix}%s%s/Undetermined/fastqc/{sample, .*?}_R1.fastq.gz" % (config['dirs']['intermediate'], config['stepnames']['Fastqc']),
+        undeterminedreverse="{prefix}%s%s/Undetermined/fastqc/{sample, .*?}_R2.fastq.gz" % (config['dirs']['intermediate'], config['stepnames']['Fastqc'])
+     shell:
+        " {input.forward}"
```

Comment:
did you ever execute this rule? I wonder why input file paths are BEFORE the program call?!

Reply:
no, I did not
I wrote the rule and i will test it when HPC is available
I want to get your feedback because this is the first rule that I will add to spike